tools/tertiary-analysis/scmap/scmap_scmap_cell.xml

```diff
@@ -34,9 +34,9 @@
     </inputs>
     <outputs>
       <data name="output_single_cell_experiment" format="rdata" />
-      <data name="closest_cells_text_file" format="csv" />
-      <data name="closest_cells_similarities_text_file" format="csv" />
-      <data name="closest_cells_clusters_csv" format="csv">
+      <data name="closest_cells_text_file" format="tsv" />
```

Comment:
My understanding is that tabular is preferred over tsv.

Reply:
I suggest that you take the problematic file and upload it to galaxy setting it to `tabular` on upload to see if you get the error.

Reply:
@pcm32 I've tried `tabular`, it worked fine.

Reply:
(and Andrey was able to reproduce the csv error with a manual upload).